shaker_sort/Shaker_sort.rb

```diff
@@ -0,0 +1,35 @@
+def shaker_sort(input, length)
```

Comment:
Fix folder name. Change it to `Shaker_Sort`.

Reply:
sir when i changed the folder name and the  i again commited my changes , 5 more files got renamed and added in this. Now i m stuck sir. Have i done any mistake?